module_evaluations/TCR-56_2025-08-25-ui-mosaic-settings.MD

```diff
@@ -0,0 +1,135 @@
+  * Zero security issues.
+  * 0% duplication.
+  **** ML browse through code for any disabled
+  * See [Rule Customization](https://dev.folio.org/guides/code-analysis/#rule-customization) details. 
+* [X] Uses [officially supported](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) build tools (3, 5, 13)
+* [X] Unit tests have 80% coverage or greater, and are based on [officially supported technologies](https://wiki.folio.org/display/TC/Officially+Supported+Technologies)[^1] (3, 4)
+  * 97.2% test coverage
+* [ ] Assigned to exactly one application descriptor within the FOLIO Community LSP Platform, specified in the Jira task for this module evaluation (3, 5)
+  * folio_mosaic-settings does belong to exactly one application descriptor -- app-mosaic (https://github.com/folio-org/app-mosaic).  However app-mosaic is not specified as part of the FOLIO community LSP Platform at the link below.  platform-lsp includes app-platform-complete, but platform complete also does not include mosaic.  (although it does include gobi, so probably should include mosaic).
+  * _The FOLIO Community LSP Platform is defined at https://github.com/folio-org/platform-lsp._
+  * _This can be evaluated by searching application descriptors across the folio-org GitHub organization (considering those applications part of the community platform) and confirming that this module only appears in the application descriptor of the one specified application._
+
+### Frontend
+
+Note: Frontend criteria apply to both modules and shared libraries. 
+
+* [X] For each consumed API `package.json` MUST include the interface requirement in the `"okapiInterfaces"` or `"optionalOkapiInterfaces"` section (3, 5)
+  * -_note: read more at https://github.com/folio-org/stripes/blob/master/doc/dev-guide.md#the-package-file-stripes-entry_
+* [X] If provided, End-to-end tests must be written in an [officially supported technology](https://wiki.folio.org/display/TC/Officially+Supported+Technologies)[^1] (3, 4)
+  * Dev team notes that no end-to-end tests are currently provided.
+  * -_note: while it's strongly recommended that modules implement integration tests, it's not a requirement_
+  * -_note: these tests are defined in https://github.com/folio-org/stripes-testing_
+* [X] Have i18n support via react-intl and an en.json file with English texts (8)
+* [X] Have WCAG 2.1 AA compliance as measured by a current major version of axe DevTools Chrome Extension (9)
+  * 0 issues reported by axe
+* [ ] Use the Stripes version of referred on the [Officially Supported Technologies](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) page[^1] (10, 16)
+  * Module uses stripes "^10.0.0".  Trillium requires 10.1 or greater.  But is Trillium our target?  The TCR instructions use to be clear on which OST to look at.  Sunflower required 10.0 or greater.  Both S and T OST pages are currently marked "Active".
```

Comment:
10.1 is not released, at the same time `^10.0.0` will include 10.10 automatically